### ISSUE TITLE
Print to PDF crashing browser

### ISSUE THREAD

**Issue by McToomin** [stolksdorf/naturalcrit#60]
I followed the steps exactly for printing to PDF. I downloaded Chrome Canary, loaded up my homebrew into it, and hit the Print View. The Print View page loads up fine. Then I actually go to Print, having already selected Print to PDF.

It will load up the preview of the print for a second or two, and then both the Print View page and the editing page crashes with the "Aw Snap!" error. Reloading the page brings the brew back up without any issues, but right now each time I've tried it's simply crashed the page. Trying to hit Print before the preview loads in case it was the actual print preview crashing it has been unsuccessful.

Thanks for all the work you've done on this tool, it really is amazing.

**Comment by stolksdorf** [comment]
How large is the brew you are trying to print? Generally the Aw, Snap error comes from a lack of memory.

**Comment by McToomin** [comment]
It's about 12 pages.

**Comment by stolksdorf** [comment]
Are you still having issues with this?

**Comment by McToomin** [comment]
No I'm not. I've printed a number of PDFs and have not experienced the
crashing issue for a long time now.

One thing is that the paper-like background of the pages as well as the
appearance of tables don't seem to carry over to PDFs. I hadn't submitted
any bug reports, but I can show you some of my PDFs if you'd like to see,
but I figured it must be a known issue.

Still love the site, I use it pretty much every day to work on my project.
Can't thank you and anyone else working on this enough.
